### PR TITLE
Keep ContentAnalysisRequest in sync with its Chromium version.

### PR DESCRIPTION
**Pull request by nancy-lan-xiao**
I think I messed up the original pull request... Starting from a clean slate to make things easier for now.

The code change is largely a copy-paste from the Chromium version, with slight change in the comment (line 64-65)

### CODE REVIEW

File: proto/content_analysis/sdk/analysis.proto [hunk 1]
```diff
@@ -60,12 +60,23 @@ message ContentData {
...
+// should always be in sync with it (https://osscs.corp.google.com/chromium/chromium/src/+/main:components/enterprise/common/proto/connectors.proto;l=87;drc=a8fb6888aff535f27654f03cd1643868ba066de9).
 message ContentAnalysisRequest {
-  // Token used to correlate requests and responses.
+  // The TokenID for Enterprise-enrolled devices.  This identifies a specific
+  // chrome instance.
+  optional string device_token = 1;
+
+  // Firebase Cloud Messaging token used to notify this client of the verdict.
+  // This identifies a specific chrome instance.
+  optional string fcm_notification_token = 2;
```
Comment (rogerta) [inline proto/content_analysis/sdk/analysis.proto:73]: Sorry my bad, I should have been clearer.  These fields (1 and 2) are used only with cloud content analysis, so let's leave the as reserved.

**Review by rogerta — APPROVED** [review]
Thanks Nancy!  One comment below to make two fields reserved, but otherwise all looks good.  Please ask Eric for a review if needed since I will be OOO on Monday.

**Review by eric-waldman — APPROVED** [review]
LGTM

**Comment by eric-waldman** [comment]
Reverted due to build errors.

```
google\waldman@WALDMAN-WIN C:\src\content_analysis_sdk>cmake --build build
Microsoft (R) Build Engine version 17.1.0+ae57d105c for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  Generating gen/content_analysis/sdk/analysis.pb.cc
  agent_base.cc
  agent_win.cc
  session_base.cc
  session_win.cc
  analysis.pb.cc
  Generating Code...
  cac_agent.vcxproj -> C:\src\content_analysis_sdk\build\Debug\cac_agent.lib
  agent.cc
C:\src\content_analysis_sdk\demo\agent.cc(51,32): error C2248: 'content_analysis::sdk::ContentAnalysisRequest::
has_content_data': cannot access private member declared in class 'content_analysis::sdk::ContentAnalysisReques 
t' [C:\src\content_analysis_sdk\build\agent.vcxproj]
C:\src\content_analysis_sdk\build\gen\content_analysis/sdk/analysis.pb.h(2597): message : see declaration of 'c 
ontent_analysis::sdk::ContentAnalysisRequest::has_content_data' [C:\src\content_analysis_sdk\build\agent.vcxpro 
j]
C:\src\content_analysis_sdk\build\gen\content_analysis/sdk/analysis.pb.h(813): message : see declaration of 'co 
ntent_analysis::sdk::ContentAnalysisRequest' [C:\src\content_analysis_sdk\build\agent.vcxproj]
C:\src\content_analysis_sdk\demo\agent.cc(51,45): error C2039: 'content_data': is not a member of 'content_anal 
ysis::sdk::ContentAnalysisRequest' [C:\src\content_analysis_sdk\build\agent.vcxproj]
C:\src\content_analysis_sdk\build\gen\content_analysis/sdk/analysis.pb.h(813): message : see declaration of 'co 
ntent_analysis::sdk::ContentAnalysisRequest' [C:\src\content_analysis_sdk\build\agent.vcxproj]
C:\src\content_analysis_sdk\demo\agent.cc(52,17): error C2039: 'content_data': is not a member of 'content_anal 
ysis::sdk::ContentAnalysisRequest' [C:\src\content_analysis_sdk\build\agent.vcxproj]
C:\src\content_analysis_sdk\build\gen\content_analysis/sdk/analysis.pb.h(813): message : see declaration of 'co 
ntent_analysis::sdk::ContentAnalysisRequest' [C:\src\content_analysis_sdk\build\agent.vcxproj]
C:\src\content_analysis_sdk\demo\agent.cc(107,6): error C2248: 'content_analysis::sdk::ContentAnalysisRequest:: 
has_content_data': cannot access private member declared in class 'content_analysis::sdk::ContentAnalysisReques 
t' [C:\src\content_analysis_sdk\build\agent.vcxproj]
C:\src\content_analysis_sdk\build\gen\content_analysis/sdk/analysis.pb.h(2597): message : see declaration of 'c 
ontent_analysis::sdk::ContentAnalysisRequest::has_content_data' [C:\src\content_analysis_sdk\build\agent.vcxpro 
j]
  queued_agent_win.cc
...
```

**Pull request merged**